src/types/form.ts

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash'
```

Comment:
prefer importing just the specific function so the frontend doesn't have to load the entire `lodash` library, which is huge. I _think_ this is the syntax:
```suggestion
import range from 'lodash/range'
```